src/components/layouts/landing-inner.njk

```diff
@@ -15,8 +15,8 @@
   {% render '@graphic-list' %}
 
   <section class="usa-section">
-    <div class="usa-grid">
-      <h2>Section heading</h2>
+    <div class="grid-container">
```

Comment:
I feel like `usa-prose` _should_ work here, but it isn't. I'm not getting the `heading + *` spacing and the button text color is being overridden by the prose class...

<img width="989" alt="screen shot 2018-09-20 at 10 21 19 am" src="https://user-images.githubusercontent.com/11464021/45835346-adc55900-bcbe-11e8-8cae-95a07206274e.png">

I'll need to dig into this a bit more

Reply:
Just a note to self to address this in a subsequent issue.
https://github.com/uswds/uswds/issues/2758